sql/core/src/test/resources/sql-tests/inputs/window.sql

```diff
@@ -465,3 +465,60 @@ SELECT
 	SUM(salary) OVER w sum_salary
 FROM
 	basic_pays;
+
+-- Test QUALIFY clause
```

Comment:
We can split these tests into a new file.

Reply:
OK